data/campaigns/Eastern_Invasion/scenarios/11_Captured.cfg

```diff
@@ -15,10 +15,10 @@
     # wmllint: local spelling Dra-Nak
     [story]
         [part]
-            story= _ "After reaching what they believed was safety, Gweddry and his men had been captured by northern orcs."
+            story= _ "After reaching what they believed was safety, Gweddry and his men had been captured by trolls from the northern orcish tribes."
```

Comment:
There are no Trolls in Orcish tribes.

Come on, spend another 2 minutes on phrasing some piece of narrative relating the trolls to the Orcs.

Reply:
If you're not content with the current patch, then you can make your own pull requests with _your_ proposed changes.